packages/samples/headless-react/src/pages/AtomicReactPage.css

```diff
@@ -0,0 +1,168 @@
+atomic-color-facet[field='cat_color']::part(value-Black) {
```

Comment:
If someone is looking to use Atomic React, would they need to copy this style sheet? Or is it just for our default layout and theme?

Reply:
Well they would need it if they want to have this specific spacing/grids etc. Which is no different than the basic Atomic lib today.

If you remember there's a thing we want to do in 2022 for Atomic, which is to provide some "layouts" components, for predefined use cases.

So for examples, we would have "default commerce layouts" components, which would drastically remove the need for all this CSS if you want to have something that looks exactly like our default pages.

Does that make sense ?

Reply:
Sounds good. In the short term, should we extract the layout styles into a CSS file and reference it inside Atomic and here so that the layouts stay in sync? or is it better to keep separate copies?